src/test/java/tests/ConformanceTest.java

```diff
@@ -209,7 +209,12 @@ protected String getFactText() {
         return cannotConvert(sourceType, sinkType);
       }
       if (IRRELEVANT_ANNOTATION_KEYS.contains(detailMessage.messageKey)) {
-        return irrelevantAnnotation("Nullable"); // TODO(dpb): Support other annotations.
+        if (detailMessage.messageArguments.isEmpty()) {
+          // arguments must start with the annotation
+          return toString();
+        }
+        return irrelevantAnnotation(
+            detailMessage.messageArguments.get(0).replaceFirst(".*\\.", ""));
```

Comment:
What is this replacing?

Reply:
It's removing the package name and leaving just the simple name.

Reply:
Hmm. Are there other places, either in this plugin or in the framework itself, where an annotation's name is included in an error or warning? Where else is an error reported when an annotation is applied incorrectly? I think this plugin should follow whatever the existing practice is.

I think the fully-qualified name is useful to report in case there is more than one annotation with the same simple name in scope.

For types that are reported, it appears that simple names are reported, so you get `List<Foo>` and not `java.util.List<some.pkg.Foo>`. I had assumed that it was not reporting simple names _per se_, but whatever name is in scope.

I don't want to perturb the checker too much just to make the output match the conformance test comment. I think there's a valid reason to do some of that in the test integration.